src/github.com/matrix-org/dendrite/clientapi/routing/routing.go

```diff
@@ -100,6 +100,13 @@ func Setup(
 			return writers.SendEvent(req, device, vars["roomID"], vars["eventType"], vars["txnID"], &emptyString, cfg, queryAPI, producer)
 		}),
 	)
+	r0mux.Handle("/rooms/{roomID}/state/{eventType}/",
```

Comment:
Indeed I could use a regular expression instead of just a variable name. I'm still exchanging a bit in the gorilla repo to see if there's a better way, but if not I'll go for that.